File: VB/DXApplication1/DataAccess/ConnectionSettings.vb
Imports System
Imports System.Collections.Generic
Imports System.Data.SqlClient
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DXApplication.Data
	Public Module ConnectionSettings
		Public DataSource, UserID, Password, InitialCatalog As String
		Public SelectQuery As String
		Public Sub SetUp(ByVal dataSource As String, ByVal userId As String, ByVal password As String, ByVal initialCatalog As String)
			ConnectionSettings.DataSource = dataSource
			ConnectionSettings.UserID = userId
			ConnectionSettings.Password = password
			ConnectionSettings.InitialCatalog = initialCatalog
		End Sub
		Public Function GetConnectionString() As String
			Dim builder = New SqlConnectionStringBuilder()
			builder.DataSource = ConnectionSettings.DataSource
			builder.UserID = ConnectionSettings.UserID
			builder.Password = ConnectionSettings.Password
			builder.InitialCatalog = ConnectionSettings.InitialCatalog
			Return builder.ConnectionString
		End Function
	End Module
End Namespace

File: CS/DXApplication1/DataAccess/ConnectionSettings.cs
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DXApplication.Data {
    public static class ConnectionSettings {
        public static string DataSource, UserID, Password, InitialCatalog;
        public static string SelectQuery;
        public static void SetUp(string dataSource, string userId, string password, string initialCatalog) {
            DataSource = dataSource;
            UserID = userId;
            Password = password;
            InitialCatalog = initialCatalog;
        }
        public static string GetConnectionString() {
            var builder = new SqlConnectionStringBuilder();
            builder.DataSource = ConnectionSettings.DataSource;
            builder.UserID = ConnectionSettings.UserID;
            builder.Password = ConnectionSettings.Password;
            builder.InitialCatalog = ConnectionSettings.InitialCatalog;
            return builder.ConnectionString;
        } 
    }
}
